src/components/timetable/view/TiemtableViewColumn.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+import { useTheme } from '@emotion/react';
+import styled from '@emotion/styled';
+
+import { Moment } from 'moment';
+
+import TimetableCell from './TimetableCell';
+import { TIMETABLE_CELL_MIN_WIDTH } from '@/styles/timetable';
+import {
+  TIMETABLE_CELL_AMOUNT,
+  TIMETABLE_CELL_PER_HOUR,
+} from '@/utils/constants';
+
+type TProps = {
+  date: Moment;
+};
+
+const TiemtableViewColumn: React.FC<TProps> = ({ date }) => {
+  const theme = useTheme();
+
+  return (
+    <Container>
+      {Array.from(Array(TIMETABLE_CELL_AMOUNT), (_, index) => {
+        const cellStyle: { borderBottom?: string } = {};
+        if (Math.floor(index % TIMETABLE_CELL_PER_HOUR) === 3) {
+          cellStyle.borderBottom = `1px solid ${theme.border2}`;
+        }
+        return <TimetableCell css={cellStyle}></TimetableCell>;
```

Comment:
4번째 칸에 `borderBottom`을 주는 코드 맞나요? 그렇다면 `Container` 컴포넌트에 
```suggestion
const Container = styled.div`
...
    &:nth-child(4n) {
        border-bottom: `px solid ${({ theme }) => theme.border2};
    }
`;
```

을 추가하는 방법은 안될까요?

Reply:
`TimetableCell`에 해당 내용을 추가했습니다!